etc/skydive.yml.default

```diff
@@ -105,12 +107,15 @@ agent:
   # X509_servername: domain.com
   #
   http:
-    # log the HTTP client request and response (to log level DEBUG)
-    # debug: false
     # define the Cookie HTTP Request Header
     # cookie: 
     #   <name1>: <value1>
     #   <name2>: <value2>
+    # rest:
+    #   log the HTTP client request and response (to log level DEBUG)
+    #   debug: false
+    # websocket:
```

Comment:
we already have a bunch of Websocket parameter at the root level maybe we could place this close to them (ws_enable_writre_compression)

Reply:
will move it to the ws_ block, but have renamed the entire block from `ws_*` to `http.ws.*`